testing/src/Environments.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Temporal\Testing;
+
+final class Environments
```

Comment:
Hi @cv65kr.
I am not sure that class is needed.
We can replace it with closure that will stop all Environment objects.
What do you think?

Reply:
@roxblnfk fixed

Reply:
@roxblnfk fixed